tests/integration/models/reddit/test_comment.py

```diff
@@ -260,6 +260,23 @@ def test_remove(self):
                 'TestCommentModeration.test_remove'):
             self.reddit.comment('da2g5y6').mod.remove(spam=True)
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_send_removal_message(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestCommentModeration.test_send_removal_message'):
+            comment = self.reddit.comment('edu698v')
+            mod = comment.mod
+            mod.remove()
+            message = 'message'
+            res = [mod.send_removal_message(t, 'title', message)
```

Comment:
What's the behavior of sending three removal messages for the same message? Does it add a comment, then send two messages?

Reply:
Yes, all of the messages are sent.